prepare.sh

```diff
@@ -0,0 +1,156 @@
+DATA_SETS_FA=(
+  "all-clean-fa/GlobalVoices.en-fa"
+  "all-clean-fa/GNOME.en-fa"
+  "all-clean-fa/infopankki.en-fa"
+  "all-clean-fa/KDE4.en-fa"
+  "all-clean-fa/OpenSubtitles.en-fa"
+  "all-clean-fa/QED.en-fa"
+  "all-clean-fa/Tanzil.en-fa"
+  "all-clean-fa/TED2013.en-fa"
+  "all-clean-fa/TEP.en-fa"
+  "all-clean-fa/Ubuntu.en-fa"
+  "all-clean-fa/Wikipedia.en-fa"
+)
+
+echo "Aggregating splits"
+for FILE in "${TRAIN_SETS_PS[@]}" ; do cat "${DATA}/$FILE.ps"; done > $DATA/undeduped-train.ps-en.ps
+for FILE in "${TRAIN_SETS_PS[@]}" ; do cat "${DATA}/$FILE.en"; done > $DATA/undeduped-train.ps-en.en
+for FILE in "${TRAIN_SETS_KM[@]}" ; do cat "${DATA}/$FILE.km"; done > $DATA/undeduped-train.km-en.km
+for FILE in "${TRAIN_SETS_KM[@]}" ; do cat "${DATA}/$FILE.en"; done > $DATA/undeduped-train.km-en.en
+for FILE in "${TRAIN_SETS_HI[@]}" ; do cat "${DATA}/$FILE.hi"; done > $DATA/undeduped-train.hi-en.hi
+for FILE in "${TRAIN_SETS_HI[@]}" ; do cat "${DATA}/$FILE.en"; done > $DATA/undeduped-train.hi-en.en
+for FILE in "${DATA_SETS_FA[@]}"  ; do cat "${DATA}/$FILE.fa"; done | awk 'NR % 1500 >  1' > $DATA/undeduped-train.fa-en.fa
+for FILE in "${DATA_SETS_FA[@]}"  ; do cat "${DATA}/$FILE.en"; done | awk 'NR % 1500 >  1' > $DATA/undeduped-train.fa-en.en
+for FILE in "${DATA_SETS_FA[@]}"  ; do cat "${DATA}/$FILE.fa"; done | awk 'NR % 1500 == 0' > $DATA/undeduped-valid.fa-en.fa
+for FILE in "${DATA_SETS_FA[@]}"  ; do cat "${DATA}/$FILE.en"; done | awk 'NR % 1500 == 0' > $DATA/undeduped-valid.fa-en.en
+for FILE in "${DATA_SETS_FA[@]}"  ; do cat "${DATA}/$FILE.fa"; done | awk 'NR % 1500 == 1' > $DATA/undeduped-test.fa-en.fa
+for FILE in "${DATA_SETS_FA[@]}"  ; do cat "${DATA}/$FILE.en"; done | awk 'NR % 1500 == 1' > $DATA/undeduped-test.fa-en.en
+echo "Deduplicating splits"
+dedupe () {
+  dedupe_set=$1
+  dedupe_lang=$2
+  python3 scripts/deduplicate.py \
+  --input-src  $DATA/undeduped-$dedupe_set.$dedupe_lang-en.$dedupe_lang \
+  --input-tgt  $DATA/undeduped-$dedupe_set.$dedupe_lang-en.en \
+  --output-src $DATA/$dedupe_set.$dedupe_lang-en.$dedupe_lang \
+  --output-tgt $DATA/$dedupe_set.$dedupe_lang-en.en
+}
+dedupe "train" "ps"
+dedupe "train" "km"
+dedupe "train" "hi"
+dedupe "train" "fa"
+dedupe "valid" "fa"
+dedupe "test"  "fa"
+cat "${DEVTEST_PSKM}/ps-en.dev.ps"     > $DATA/valid.ps-en.ps
+cat "${DEVTEST_PSKM}/ps-en.dev.en"     > $DATA/valid.ps-en.en
+cat "${DEVTEST_PSKM}/ps-en.devtest.ps" > $DATA/test.ps-en.ps
+cat "${DEVTEST_PSKM}/ps-en.devtest.en" > $DATA/test.ps-en.en
+cat "${DEVTEST_PSKM}/km-en.dev.km"     > $DATA/valid.km-en.km
+cat "${DEVTEST_PSKM}/km-en.dev.en"     > $DATA/valid.km-en.en
+cat "${DEVTEST_PSKM}/km-en.devtest.km" > $DATA/test.km-en.km
+cat "${DEVTEST_PSKM}/km-en.devtest.en" > $DATA/test.km-en.en
+cat "${DEVTEST_HI}/dev.hi"          > $DATA/valid.hi-en.hi
+cat "${DEVTEST_HI}/dev.en"          > $DATA/valid.hi-en.en
+cat "${DEVTEST_HI}/test.hi"         > $DATA/test.hi-en.hi
+cat "${DEVTEST_HI}/test.en"         > $DATA/test.hi-en.en
```

Comment:
`DEVTEST_PSKM` and `DEVTEST_HI` are placeholders for where these datasets will eventually live.